src/utils/make-command.js

```diff
@@ -1,7 +1,34 @@
 const { EOL } = require( 'os' );
+const chalk = require( 'chalk' );
+const logSymbols = require( 'log-symbols' );
+const makeDocker = require( '../utils/make-docker' );
 
-module.exports = function makeCommand( chalk, { error }, command ) {
+/**
+ * Check if Docker app is running.
+ *
+ * @return {Promise<boolean|*>}
+ */
+async function pingDocker() {
+    try{
+        const docker = makeDocker();
+        return await docker.ping();
+    } catch( ex ) {
+        return false;
+    }
+}
+
+module.exports = function makeCommand( options = {}, { error }, command ) {
```

Comment:
@thrijith `{ error }` here is an equivalent of `const { error } = logSymbols;`. So we don't need it and can remove from the `makeCommand` signature in favor of using `logSymbols` directly in this file. 

It will let us not to use the second parameter when we use this function in actual commands. So isntead of `makeCommand( {}, {}, async () => { ... } );` we will just need to use `makeCommand( {}, async () => {} );`.